tla-types/src/main/scala/at/forsyte/apalache/tla/types/TypeReduction.scala

```diff
@@ -34,15 +34,15 @@ class TypeReduction(private val gen: SmtVarGenerator) {
       val phi = sizeOfEql(tupIdx, l) +: constraints
       ReductionResult(tup(tupIdx), phi)
     case RecT(ts @ _*) =>
-      val l = ts.length
+      ts.length
       val recIdx = gen.getFreshInt
       val constraints = ts.flatMap { case KvPair(k, v) =>
         val ReductionResult(t, phi) = apply(v, m)
         hasField(recIdx, k, t) +: phi
       }
       ReductionResult(rec(recIdx), constraints)
     case SparseTupT(ts @ _*) =>
-      val l = ts.length
+      ts.length
```

Comment:
I guess these should be removed, unless there's a logic error by us not using these values.